pages/cloud/private-cloud/lifecycle/guide.fr-fr.md

```diff
@@ -0,0 +1,204 @@
+
+Commencez par ajouter de nouveaux datastores en suivant le guide « [Ajouter des espaces de stockage](https://docs.ovh.com/fr/private-cloud/additional-storage/) ».
+
+Utilisez ensuite la fonctionalité *Storage vMotion* de VMware pour déplacer vos VMs vers vos nouveaux datastores. Vous pouvez consulter la documentation VMware « [Storage vMotion](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vcenterhost.doc/GUID-A15EE2F6-AAF5-40DC-98B7-0DF72E166888.html) ».
+
+Enfin, supprimez vos anciens datastores en suivant le guide « [Supprimer un datastore](https://docs.ovh.com/fr/private-cloud/suppression-data-store/) ».
+
+### Cycle de vie des hosts 
+
+L'objectif de cette étape est de vérifier si certains de vos hosts nécessitent des actions liées au cycle de vie des environnements VMware hébergés chez OVHcloud.
+
+#### Récupérer votre type de datastore
+
+Exécutez l'appel suivant :
+
+> [!api]
+>
+> @api {GET} /dedicatedCloud/{serviceName}/datacenter/{datacenterId}/filer/{filerId}
+>
+
+Renseignez les champs de l'appel avec les informations suivantes :
+
+- **serviceName** : renseignez le serviceName obtenu à l'étape 1.1
+- **datacenterId** : renseignez le datacenterId obtenu à l'étape 1.2
+- **hostId** : renseignez le hostId obtenu à l'étape 1.4
+
+Si la réponse au champ **profile** est `Dedicated Cloud`, il est alors nécéssaire de passer aux nouvelles gammes. En effet, ces hosts ne sont plus compatibles avec les dernières versions d'ESXi et ne peuvent donc plus être supportés par VMware ou OVHcloud.
+
+Pour réaliser votre migration, vous pouvez suivre ce guide : [Migrer vers une nouvelle infrastructure](https://docs.ovh.com/fr/private-cloud/vdc-migration/).
+
+### Cycle de vie des composants software
+
+#### Récupérer les informations de votre PCC dans l'espace client OVHcloud
+
+Connectez-vous à l'[espace client OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/fr/&ovhSubsidiary=fr) avec un compte administrateur.
+
+Cliquez sur l'onglet `Hosted Private Cloud`{.action} et sélectionnez votre service dans la liste `VMware`{.action}.
+
+Dans l'onglet `Informations générales`{.action}, récupérez la version de votre vSphere.
+
+![espace client - récupération de la version de vsphere](images/manager-version-vsphere.png){.thumbnail}
+
+Si votre version de vSphere n'est pas vCSA 7.0 (build 20845200), veuillez créer un ticket afin de planifier une mise à jour vers la dernière version.
+
+#### Récupérer les informations de votre PCC dans vSphere
+
+Rendez vous dans la vue `Mise en réseau`{.action} de votre client vSphere puis dans `Sécurité`{.action} et enfin cliquez sur l'onglet `Upgrade`{.action}.
+
+Prenez note de la version de votre NSX-v.
+
+![vSphere - récupération des données](images/vsphere-version-nsx-v.png){.thumbnail}
```

Comment:
@Hubmar, please provide the vSphere screenshot.
I added the Control Panel screenshot, please check if it's valid.

Reply:
Screenshot and valid path provided by Naïm.